bus/client/metrics.go

```diff
@@ -70,3 +71,31 @@ func (c *Client) RecordContractSetChurnMetric(ctx context.Context, metrics ...ap
 		Metrics: metrics,
 	})
 }
+
+func (c *Client) ContractMetrics(ctx context.Context, opts api.ContractMetricsQueryOpts) ([]api.ContractMetric, error) {
+	values := url.Values{}
+	if opts.After != (time.Time{}) {
```

Comment:
`nit` I've been adding an `apply` method on `api.ContractMetricsQueryOpts` which I think is nicer